components/main.tf

```diff
@@ -49,11 +49,8 @@ resource "github_organization_ruleset" "default_ruleset" {
     required_linear_history = true
 
     pull_request {
-      dismiss_stale_reviews_on_push     = true
```

Comment:
This rule discourages pushing small changes and fixups as you have to go chasing reviews again

What was the reason this was enabled?